DotNet/LinkAdmin.BFF/Presentation/Endpoints/AuthEndpoints.cs

```diff
@@ -87,8 +89,34 @@ public IResult Login(HttpContext context)
 
         public IResult GetUser(HttpContext context)
```

Comment:
Would be good to have a comment on this public method that provides more details on what this is used for and more importantly what it returns. IResult is very generic and I wouldn't be able to tell at a glance what IResult actually is.